examples/migrate_cluster_to_advanced_cluster/module_maintainer/v3/main.tf

```
@@ -0,0 +1,69 @@
+locals {
+  old_disk_size     = var.auto_scaling_disk_gb_enabled ? null : var.disk_size
```

```suggestion
  old_disk_size     = coalesce(var.auto_scaling_disk_gb_enabled, var.disk_size)
```

nit: I think this is equivalent but maybe your code is more clear